src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonProperty.cs

```diff
@@ -45,7 +45,7 @@ internal JsonProperty(JsonElement value, string name = null)
         ///   This method is functionally equal to doing an ordinal comparison of <paramref name="text" /> and
         ///   <see cref="Name" />, but can avoid creating the string instance.
         /// </remarks>
-        public bool NameEquals(string text)
+        public bool NameEquals(string? text)
```

Comment:
Please fix this in the ref as well (and apply any other leftover feedback from https://github.com/dotnet/runtime/pull/114)

Reply:
Sure thing